### PR TITLE
Run go lint using github actions

### PR DESCRIPTION
**Pull request by MartinForReal**
#### What type of PR is this?
/kind testing

#### What this PR does / why we need it:
Run go lint using github actions
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/consts/helpers.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func extractInt32FromString(val string, businessValidator ...Int32BusinessValida
 	errKey := fmt.Errorf("%s value must be a whole number", val)
 	toInt, err := strconv.ParseInt(val, 10, 32)
 	if err != nil {
-		return nil, fmt.Errorf("error value: %w: %v", err, errKey)
```
Comment (lzhecheng) [inline pkg/consts/helpers.go:96]: CodeQL suggests "fmt.Errorf call has more than one error-wrapping directive"
Is this helpful? https://stackoverflow.com/questions/65729195/wrapping-more-than-one-static-error-to-use-with-errors-is

Comment (MartinForReal) [reply pkg/consts/helpers.go:96]: replace fmt.Error with sprint in lin 93.Thanks for the tips!

**Pull request merged**